lib/fileset/default.nix

```diff
@@ -234,34 +264,47 @@ in {
 
     This variant is useful for tracing file sets passed as arguments to other functions.
 
-    Type:
-      traceVal :: FileSet -> FileSet
-
-    Example:
-      toSource {
-        root = ./.;
-        fileset = traceVal (unions [
-          ./Makefile
-          ./src
-          ./tests/run.sh
-        ]);
-      }
-      =>
-      trace: /home/user/src/myProject
-      trace: - Makefile (regular)
-      trace: - src (all files in directory)
-      trace: - tests
-      trace:   - run.sh (regular)
-      "/nix/store/...-source"
-  */
-  traceVal =
-    /*
-    The file set to trace and return.
+
+    # Inputs
+
+    `fileset`
+
+    : The file set to trace and return.
 
     This argument can also be a path,
     which gets [implicitly coerced to a file set](#sec-fileset-path-coercion).
```

Comment:
This content is for the same argument. The last 2 lines should have two extra spaces of indentation so it's rendered correctly.